.github/workflows/test.yaml

```
@@ -26,7 +26,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-12
+          # - macos-12
           - macos-latest
```

```suggestion
          # - macos-latest
```

since unlikely to work there either